cleancat/base.py

```diff
@@ -122,7 +122,15 @@ def _check_length(self, value):
             raise ValidationError(err_msg)
 
     def clean(self, value):
-        value = super(String, self).clean(value)
+        try:
+            value = super(String, self).clean(value)
+        except ValidationError as exc:
+            # This is a layer to keep the external API unchanged once we
+            # migrate to Python 3.
+            if six.PY3 and str(exc) == 'Value must be of str type.':
+                raise ValidationError('Value must be of basestring type.')
```

Comment:
There's no `basestring` in Py3. `... must be a string or bytes`. Do we allow bytes? If not, just `a string`.

Reply:
There are more `basestring`s below but I didn't comment on them all.

Reply:
Done in https://github.com/closeio/cleancat/pull/52/commits/6cb639855462571d75d40a5712352354aed669ef.